rskj-core/src/test/java/org/ethereum/db/IndexedBlockStoreTest.java

```diff
@@ -92,8 +93,8 @@ public void setup() throws URISyntaxException, IOException {
     @Test // save some load, and check it exist
     @Ignore
     public void test1(){
-
-        IndexedBlockStore indexedBlockStore = new IndexedBlockStore(ConfigHelper.CONFIG);
+        config = new RskSystemProperties();
```

Comment:
I think we already have these in the `@Before` step

Reply:
Yes I forgot that I moved the instance creation to @Before. I'll remove all the creations inside the tests :+1: